components/bl-double-calendar-component/src/specific-date-range.js

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+
+import { addDays } from './lib/date-fns.min.js';
+import DatePicker from './lib/react-datepicker.min.js';
+
+export function SpecificDateRange({ selectionRange, defaultSelectedDate, onDateSelect }) {
+  const [selectedDate, setSelectedDate] = useState(defaultSelectedDate ? new Date(defaultSelectedDate) : new Date());
+
+  const handleDateSelect = date => {
+    setSelectedDate(date);
+
+    if (onDateSelect) {
+      onDateSelect({ selectedDate: date });
+    }
```

Comment:
`onDateSelect && onDateSelect({ selectedDate: date });`

@Valodya What if we use this notation for handler cases where logic every time the same?

Reply:
@v-excelsior no, this is a bad way, logic operator we use when we assign a result

saving 2 line does not make any difference